tools/src/css_analyzer.rs

```diff
@@ -0,0 +1,163 @@
+use cssparser;
+use ustr::{Ustr, ustr};
+
+use crate::file_format::analysis::{
+    AnalysisKind, AnalysisSource, AnalysisTarget, LineRange, Location, SourceRange, SourceTag, TargetTag, WithLocation
+};
+
+// NOTE: This does the same as analysis_manglings::mangle_file without regex
+//       dependency.  regex increases the wasm file size by ~80kB.
+fn mangle_name(name: &str) -> String {
+    let mut s = String::new();
+
+    for c in name.bytes() {
+        match c {
+            b'A'..=b'Z' | b'a'..=b'z' | b'0'..=b'9' | b'_' | b'/' => {
+                s.push(c as char);
+            }
+            _ => {
+                s.push_str(format!("@{:02X}", c as u8).as_str());
+            }
+        }
+    }
+
+    s
+}
```

Comment:
mangling is not shared because of the increase in the file size of wasm binary, caused by regex, which affects the load time.
  * without regex: 108kB
  * with regex: 890kB